src/rvoice/fluid_iir_filter.c

```diff
@@ -47,6 +50,12 @@ void
 fluid_iir_filter_apply(fluid_iir_filter_t* iir_filter,
                        fluid_real_t *dsp_buf, int count)
 {
+  if(iir_filter->type == FLUID_IIR_DISABLED || iir_filter->q_lin == 0)
```

Comment:
Why does setting q_lin to 0 disable the filter?

And it seems like this would not only apply to the custom filter, but also to the standard low-pass. Is it according to SD2 spec that the filter switches off if q_lin = 0?